packages/react-devtools-extensions/package.json

```diff
@@ -25,7 +25,6 @@
     "archiver": "^3.0.0",
     "babel-core": "^7.0.0-bridge",
     "babel-eslint": "^9.0.0",
-    "babel-jest": "^24.7.1",
```

Comment:
these are just remnants after migrating it into this monorepo, I guess? They're unused